ArduCopter/ArduCopter.cpp

```diff
@@ -143,6 +143,9 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 #if CAMERA == ENABLED
     SCHED_TASK_CLASS(AP_Camera,            &copter.camera,              update_trigger,  50,  75),
 #endif
+#if VISCA == ENABLED
```

Comment:
Replace `VISCA` with `VISCA_ENABLED`
Add '//#define VISCA_ENABLED DISABLED` in the APM_Copter.h file

Reply:
I updated that already, but I don't know how to mark as solved this particular changes you requested. I am working on the rest of the PR, I hope to get it on the beginning of the next week.

Thank you.